script.js

```diff
@@ -17,3 +17,191 @@ menuArr.forEach((element) => {
+  toolsData.forEach((tool) => {
+    tools += `
+    <button class="link">${tool}</button>
+    `;
+  });
+  return tools;
+};
+
+// portfolio template
+const workCard = (portfolioData) => {
+  let content = '';
+  portfolioData.forEach((workData) => {
+    content += `
+  <div class="mainContainer1 mobile-cards">
+  <div class="img">
+    <img class="imgBG" src="${
+  workData.image
+}" alt="Catchy look inspiring bionic forms"></img>
+  </div>
+  <div class="infoCard">
+    <section class="info">
+      <h2 class="name">${workData.name}</h2>
+      <ul class="info_job">
+        <li class="job">CANOPY</li>
+        <li>${workData.stack}</li>
+        <li>${workData.year}</li>
+      </ul>
+    </section>
+    <section class="decription">
+      <p>${workData.excerpt}</p>
+    </section>
+    <section class="linkButtons">${toolsTemplate(workData.tools)}</section>
+    <div class="seeMore">
+      <button class="more portfolioButton" data-id="${
+  workData.id
+}">See Project</button>
+    </div>
+  </div>
+  </div>`;
+  });
+  return content;
+};
+
+// modal pop up Template
+const modal = (workData) => `<section class="popup">
+  <div class="mobile-body">
+    <div class="popupCard">
+            <section class="info">
+              <div class="icon">
+              <h2 class="name">${workData.name}</h2>
+              <a href="#" class="iconX toggleNav">
+                <img src="img/Icon.png" alt="menu_exit_button" />
+              </a>
+            </div>
+              <ul class="info_job">
+                <li class="job">CANOPY</li>
+                <li>${workData.stack}</li>
+                <li>${workData.name}</li>
+              </ul>
+            </section>
+            <div class="img">
+              <img class="imgBG" src="${
+  workData.image
+}" alt="Image from the article for Tanner Christesen"></img>
+            </div>
+            <div class="decription">
+            <div class="decriptionLeft">
+              <p class="${workData.description}">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+              </p>
+            </div>
+            <div class="decriptionRight">
+            <section class="linkButtons">${toolsTemplate(
+    workData.tools,
+  )}</section>
+            <div class="seeMore btnPopupDiv">
+              <a href="${
+  workData.live_link
+}" class="more btnPopup">See live <img class="" src="img/live.png" alt="go live icon"></img></a>
+              <a href="${
+  workData.source_link
+}"class="more btnPopup">See Source <img class="" src="img/git.png" alt="go live icon"></img></a>
+            </div>
+          </div>
+        </div>
+  </div>
+  </div>
+</section>`;
```

Comment:
You have done a great job with your modal popup so far! There are a few design related pointers left to make your project perfect.

- [ ] **On desktop view** when the popup is opened, I am not able to scroll down and see the lower side of the popup i.e. the buttons and some of the bottom sections are not able to be seen.

**Below is how I am seeing the desktop view right now**

[Screencast from 21-12-2022 03:21:09 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/208903941-98a5a7b2-c118-4b29-86c3-dc54cffbdefa.webm)

**This is the desired desktop view**

![image](https://user-images.githubusercontent.com/86318284/208904043-c3a0e7b0-ae80-4a2b-9d0c-8c7d30f223d3.png)


- [ ] **On mobile view** The trend is almost similar to the one for the desktop view. The lower section of the popup is not fully visible. Also, there is supposed to be a padding/ margin all around the popup. Kindly implement it.

**This is how the mobile view looks like:**

![image](https://user-images.githubusercontent.com/86318284/208905167-62f5ba60-4901-4da5-8e4f-97acba6e3351.png)

**This is the desired mobile view:**

![image](https://user-images.githubusercontent.com/86318284/208905304-a471dcad-1390-4f86-ba44-a0ca9ebb4987.png)